.github/workflows/deploy.yaml

```diff
@@ -26,12 +28,16 @@ jobs:
         id: meta
         uses: docker/metadata-action@v5
         with:
+          flavor:
+            # don't automatically tag with `latest`; we do this conditionally in the `tags` section
+            latest=false
           images: |
             ${{ vars.DOCKERHUB_REPO }}
             ${{ vars.GAR_LOCATION }}-docker.pkg.dev/${{ vars.GCP_PROJECT_ID }}/${{ vars.GAR_REPOSITORY}}/autograph
           tags: |
-            type=semver,pattern={{raw}}
```

Comment:
I believe removing this will break the automated AWS stage deployment pipeline. It looks for pushes to a semver-patterned docker image.

Reply:
ah, ty. That's another argument for using something suffixed for GCP. I'll fix this up.